vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj.filters

```diff
@@ -670,6 +675,7 @@
     <ClInclude Include="Views\ReactRootControl.h">
       <Filter>Views</Filter>
     </ClInclude>
+    <ClInclude Include="Modules\AsyncStorageModuleWin32.h">
```

Comment:
this and the win32 cpp should not be in the main project if it's win32 only.

Reply:
It can't be moved to the Desktop projects, because it demands Win10 or later.
It also doesn't justify a full static library project, IMHO.

Should we go ahead and delete it, since it's not relevant for RN 0.62?

Reply:
if it's not needed, yes please delete :)

Reply:
My pleasure!